lib/tasks/factory_girl.rake

```diff
@@ -0,0 +1,22 @@
+namespace :factory_girl do
+  desc "Verify that all FactoryGirl factories are valid"
+  task :lint => :environment do
+    if Rails.env.test?
+      begin
+        require "database_cleaner"
+        DatabaseCleaner.start
+        require Rails.root.join("spec", "support", "factory_girl_helper.rb")
+
+        factories_to_lint = FactoryGirl.factories.reject do |factory|
+          factory.name =~ /rr_pending_change/
+        end
+
+        FactoryGirl.lint factories_to_lint
+      ensure
+        DatabaseCleaner.clean
+      end
+    else
+      system("bundle exec rake factory_girl:lint RAILS_ENV='test'")
```

Comment:
You can do this just by setting ENV directly up above.  See how we do it for other tasks.